homeassistant/components/unifiprotect/light.py

```diff
@@ -42,7 +42,10 @@ async def _add_new_device(device: ProtectAdoptableDeviceModel) -> None:
         ):
             async_add_entities([ProtectLight(data, device)])
 
-    async_dispatcher_connect(hass, _ufpd(entry, DISPATCH_ADOPT), _add_new_device)
+    unsub = async_dispatcher_connect(
+        hass, _ufpd(entry, DISPATCH_ADOPT), _add_new_device
+    )
```

Comment:
```suggestion
    entry.async_on_unload(async_dispatcher_connect(
        hass, _ufpd(entry, DISPATCH_ADOPT), _add_new_device
    ))
```

Reply:
I guess since that makes it split onto another line it does not matter if chained like that.

Reply:
Oh wait, no, I see what you are doing, that is probably even better.